terms.html

```diff
@@ -92,13 +93,13 @@
   </dd>
   <dt><dfn data-lt="issuers|issuer's">issuer</dfn></dt>
   <dd>
-An <a>entity</a> that asserts <a>claims</a> about <a>subject</a>, 
-creates a <a>verifiable credential</a> from these claims,
+A roles where an <a>entity</a> asserts <a>claims</a> about one or more
```

Comment:
None of these role definitions seem to be ... definitive. Rather, they seem to describe one thing that is happening when the role is being played by an entity (i.e. "a role where such and such is going on" ... but what else? anything? is "such and such" about the role or does it "such and such" just always happen to coincide with when the role is being performed?).

I think this could be improved by changing:

"A role where an entity asserts claims..."

To:

"A role an entity may perform by asserting claims..."

This makes it clearer that when an entity is "asserting claims...", it is performing the role of "issuer".

Reply:
Changed all the definitions.